Code/Tools/ProjectManager/Resources/ProjectManager.qss

```diff
@@ -35,19 +35,33 @@ QTabBar::tab:text {
     text-align:left;
 }
 
-#createGemTab QTabBar::tab {
-    background-color: #444444;
-    font-size:16px;
-    min-height:230px;
-    margin: 0px;
-    text-align:left;
+QRadioButton#createAGemLHSTab::indicator{
+    image: url(:/nothing.png);
+}
+
+QRadioButton#createAGemLHSTab{
+    font-size: 16px;
+    height: 40px;
+    min-width: 180px;
+    spacing: 10px;
+    color: #6b6b6b;
+}
+
+
+QRadioButton#createAGemLHSTab:enabled{
+    color: #aaaaaa;
 }
 
-QTabWidget#createGemTab::tab-bar {
-    left: 0px;
+
+QRadioButton#createAGemLHSTab:checked{
+    background-color: #333333;
+    spacing: 30px;
+    color: #FFFFFF;
 }
```

Comment:
Rather than assigning anobject name to every radio button, you can use a selector based on their parent like this:
```
#createAGemLHS QRadioButton {
}
#createAGemLHS QRadioButton::indicator {
}
etc...
```